pkg/features/kube_features.go

```diff
@@ -1197,7 +1197,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 
 	genericfeatures.APIResponseCompression: {Default: true, PreRelease: featuregate.Beta},
 
-	genericfeatures.ValidatingAdmissionPolicy: {Default: false, PreRelease: featuregate.Alpha},
+	genericfeatures.ValidatingAdmissionPolicy: {Default: false, PreRelease: featuregate.Beta},
```

Comment:
some items on the checklist are still open.  Unless the PR solves all of them you may be better off to  leave this one line out.

Reply:
Captured the work for GA here: https://github.com/kubernetes/enhancements/pull/4132
Thank you!